cli/towerSysUtil.mli

```diff
@@ -43,4 +43,8 @@ module Dir : sig
     string list -> [> `Error of bool * string ] option
 
   val name : string -> string
+
+  (* Should this be something better? Bool? Error of bool * string? *)
```

Comment:
It could and maybe it should but this is probably fine for now. It would be nice to document the fact that `Unix.Unix_error`s might bubble out.